spec/unit/cli_spec.rb

```diff
@@ -660,6 +660,21 @@
   end
 
   describe "#error_handling" do
+    context "OptionParser" do
+      option_parser_exceptions= [ OptionParser::MissingArgument, OptionParser::AmbiguousOption]
```

Comment:
Surrounding space missing for operator '='.<br>Space inside square brackets detected.